File: Sakila_Net/VB_SakilaLib/Neg/NegFilmCategory.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegFilmCategory
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOFilmCategory.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( filmCategory as ClFilmCategory) as Boolean
                        Return DAO.DAOFilmCategory.SelectOne(filmCategory)     
                    End Function
                    Public  Shared Function crear( filmCategory as ClFilmCategory) as Integer
                        Return DAO.DAOFilmCategory.SqlInsert(filmCategory)     
                    End Function
                    Public  Shared Function eliminar( filmCategory as ClFilmCategory) as Integer
                        Return DAO.DAOFilmCategory.SqlDelete(filmCategory)     
                    End Function
                    Public  Shared Function actualizar( filmCategory as ClFilmCategory) as Integer
                        Return DAO.DAOFilmCategory.SqlUpdate(filmCategory)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegFilmCategory.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegFilmCategory
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOFilmCategory.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClFilmCategory filmCategory) 
        {
            return DAO.DAOFilmCategory.sqlSelect(filmCategory);
        }
        public  static int crear(ClFilmCategory filmCategory)
        {
            return DAO.DAOFilmCategory.sqlInsert(filmCategory);
        }
        public  static int eliminar(ClFilmCategory filmCategory)
        {
            return DAO.DAOFilmCategory.sqlDelete(filmCategory);
        }
        public  static int actualizar(ClFilmCategory filmCategory)
        {
            return DAO.DAOFilmCategory.sqlUpdate(filmCategory);
        }
    }
}
